src/storage/operations/stat.rs

```diff
@@ -0,0 +1,51 @@
+use crate::error::Result;
+use opendal::{EntryMode, Operator};
+
+#[derive(Debug, Clone)]
+pub struct ObjectMeta {
+    pub path: String,
+    pub entry_type: String, // file | dir | other
+    pub size: u64,
+    pub last_modified: Option<String>,
+    pub etag: Option<String>,
+    pub content_type: Option<String>,
+}
+
```

Comment:
We need to add Rust documentation comments, e.g. `/// Features for retrieving object metadata from the object store`. You can refer to @`list.rs`.

Reply:
OK